ManualSetup/6_InsertData_USER_KNIGHTS_RANK.sql

```diff
@@ -0,0 +1,101 @@
+INSERT INTO [USER_KNIGHTS_RANK] (shIndex, strName, strElmoUserID, strElmoKnightsName, nElmoLoyalty, strKarusUserID, strKarusKnightsName, nKarusLoyalty, nMoney) VALUES
+(1, N'Gold Knight          ', N'test                 ', N'test                 ', 50000000, NULL, NULL, 0, 1000000),
```

Comment:
USER_KNIGHTS_RANK.strName, USER_KNIGHTS_RANK.strElmoUserID, USER_KNIGHTS_RANK.strElmoKnightsName, USER_KNIGHTS_RANK.strKarusUserID, UESR_KNIGHTS_RANK.strKarusKnightsName should be trimmed like they were in my export.